src/validators/schema.js

```diff
@@ -0,0 +1,9 @@
+const Joi = require('joi');
+
+const signupSchema = Joi.object({
```

Comment:
```suggestion
const signUpSchema = Joi.object({
```
Keep the same naming in the all files